src/rtsp/conversions/rtsp_to_hls.md

```diff
@@ -0,0 +1,9 @@
+# RTSP to HLS
+
+> Some of the terms used here are explained in detail in the section [RTSP to WebRTC](./rtsp_to_webrtc.md). Refer there if you're lost.
+
+RTSP to HLS conversion is relatively simple. The stream has to be extracted from RTP packets anyway, so there's no point
+trying to inject parameters into the stream, it's easier to pass them on externally down the pipeline to the H264 parser.
```

Comment:
Remove reference to pipeline and H264 parser. Instead, we can write that the process of unpacking RTP removes sequence numbers and we can add SPS and PPS to each H264 keyframe